inference/sources/fly/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/caarlos0/env/v6"
+	"github.com/dghubble/sling"
+	. "github.com/vlab-research/vlab/inference/inference-data"
+)
+
+func handle(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+type GetResponsesParams struct {
+	PageSize int    `url:"page_size"`
+	After    string `url:"after,omitempty"`
+}
+
+type GetResponsesResponse struct {
+	TotalItems int `json:"total_items"`
+	PageCount  int `json:"page_count"`
+	Items      []struct {
+		Parent_surveyid  string `json:"parent_surveyid"`
+		Parent_shortcode string `json:"parent_shortcode"`
+		Surveyid         string `json:"surveyid"`
+		Flowid           string `json:"flowid"`
+		Userid           string `json:"userid"`
+		Question_ref     string `json:"question_ref"`
+		Question_idx     string `json:"question_idx"`
+		Question_text    string `json:"question_text"`
+		Response         struct {
+			Text string `json:"user_agent"`
+		} `json:"res"`
+		Timestamp string `json:"timestamp"`
+		Metadata  struct {
+			UserAgent string `json:"user_agent"`
```

Comment:
I don't believe these are actually in Fly. What you'll probably have is a set of key/value pairs that you don't know the names of ahead-of-time.